eng/Versions.props

```diff
@@ -124,8 +124,17 @@
     <!-- Not auto-updated. -->
     <MicrosoftDiaSymReaderNativeVersion>1.7.0</MicrosoftDiaSymReaderNativeVersion>
     <SystemCommandLineVersion>2.0.0-beta1.20253.1</SystemCommandLineVersion>
+    <CommandLineParserVersion>2.2.0</CommandLineParserVersion>
     <NETStandardLibraryRefVersion>2.1.0</NETStandardLibraryRefVersion>
     <NetStandardLibraryVersion>2.0.3</NetStandardLibraryVersion>
+    <XunitPerformanceApiPackageVersion>1.0.0-beta-build0015</XunitPerformanceApiPackageVersion>
+    <MicrosoftDiagnosticsTracingTraceEventPackageVersion>2.0.49</MicrosoftDiagnosticsTracingTraceEventPackageVersion>
+    <MicrosoftDiagnosticsToolsRuntimeClientVersion>1.0.4-preview6.19326.1</MicrosoftDiagnosticsToolsRuntimeClientVersion>
+    <MicrosoftDiagnosticsNETCoreClientVersion>0.2.61701</MicrosoftDiagnosticsNETCoreClientVersion>
+    <!-- Scenario tests install this version of Microsoft.NetCore.App, then patch coreclr binaries via xcopy. At the moment it is
+         updated manually whenever breaking changes require it to move forward, but it would be nice if we could update it automatically
+         as we do with many of the package versions above -->
+    <BaselineMicrosoftNetCoreAppPackageVersion>2.1.0-preview3-26416-01</BaselineMicrosoftNetCoreAppPackageVersion>
```

Comment:
It is hard to believe that we did not needed to update this since `2.1.0-preview3`. It suggests that none of this is used, and it would be better to delete it instead of moving it around.

Reply:
Both JitBench projects reference it directly. I don't much about these projects...

https://github.com/dotnet/runtime/blob/95109a67ed952d51da42265d393191d5d5e322e3/src/tests/performance/Scenario/JitBench/JitBench.csproj#L36
https://github.com/dotnet/runtime/blob/95109a67ed952d51da42265d393191d5d5e322e3/src/tests/performance/Scenario/JitBench/unofficial_dotnet/JitBench.csproj#L43

Reply:
@AndyAyersMS Is JitBench (or the baseline feature of  JitBench) still used for anything?

@ViktorHofer For this PR, I would keep BaselineMicrosoftNetCoreAppPackageVersion where it is for this PR, including the comment. If it is still actively used and we decide to move it here, it will need to be also renamed to make it clear that this is version is JitBench specific.